### PR TITLE
natalia post

### PR DESCRIPTION
**Pull request by ndittren**

### CODE REVIEW

File: content/articles/jira-web.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The creation of a custom issue involved multiple steps. To accomplish this, yes you do need to create each individula custom field you would want to use in your issue. But it also involves the creation of an Issue Type, which is then attached to an Issue Type Scheme. From there, you must create a Screen and attach the newly configured custom fields. Lastly, it also requires the configuration of an Issue Type Screen Scheme.
+
+The use of schemes and screens in Jira was something that I had to research quite a bit to fully understand. Fortunately there were a lot of beginner's guides to Jira as well as a pretty active Jira community board for the questions that kept coming to me. One also had to be careful about which guide to follow, as some workflows/features are only available on Jira Cloud vs Jira Software.
+
+The main takeaway for me was the expanisveness of Jira and how many things actually go into making a complex project management tool. I had previously written a small application management system, which had a different goal ofcourse, but I can see where I can go back and improve. I look forward to learning more about Jira and management tools in general.
```
Comment (sdreher) [inline content/articles/jira-web.md:25]: space needed between "ofcourse"

Comment (zmustapha) [reply content/articles/jira-web.md:25]: I fixed this one

**Comment by zmustapha** [comment]
I added the images, and an author entry for you. And I also added line breaks for content.

File: content/articles/jira-web.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Jira Web"
+date: "2020-07-19"
+type: "post"
+authors: ["dittren"]
+tags: [jira]
+lede: "Jira net"
```
Comment (zmustapha) [inline content/articles/jira-web.md:7]: Add a short summary, or pitch of your post, max about 30 words. This summary will appear on the front page, as well as the Articles page. The thing that helps me write a summary  for my post is the answer to this question, "What is my post about?"

File: content/articles/jira-web.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+shortlede: "Jira net"
```
Comment (zmustapha) [inline content/articles/jira-web.md:8]: Whatever you have for lede, you can put that here too.

File: content/articles/jira-web.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+thumbnail: "thumbnail-jira-web.jpg"
+socmediaimg: "socmediaimg-jira-web.jpg"
+hiliteimg: "poster-jira-web.jpg"
+poster_sourceurl: "https://unsplash.com/photos/J7rRzjba-kY"
+poster_source: "Ariana Suárez on Unsplash"
+bookendanimal: "cat"
+---
+
+One of my first tasks at the CTL was to look into a more efficient way for the
+Media Team to get the information they needed when an Event requires photo
```
Comment (zmustapha) [inline content/articles/jira-web.md:19]: "information they need when an event requires"

File: content/articles/jira-web.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+and/or video. After some brainstorming, I wanted to approach the problem in a
+way that would be the least distruptive to the current workflow. I settled on
+creating a customized issue for photo/event purposes.
+
+This decision began my deep dive into Jira. Before Jira, I had some experience
+with project management tools like Trello and Asana but didn’t know much about
+the administrator tasks. At first, I imagined the work would mostly consist of
+creating custom fields to place on a template but I came to find that there was
```
Comment (zmustapha) [inline content/articles/jira-web.md:27]: "creating custom fields to place on a template, but I came to find"

File: content/articles/jira-web.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+a lot more to it. Jira is a much more complex tool than I was expecting and I
+found myself in a bit of a web.
+
+The creation of a custom issue involved multiple steps. To accomplish this, yes
```
Comment (zmustapha) [inline content/articles/jira-web.md:31]: "To accomplish this, yes,"

File: content/articles/jira-web.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+you do need to create each individula custom field you would want to use in
```
Comment (zmustapha) [inline content/articles/jira-web.md:32]: typo - "individual"

File: content/articles/jira-web.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+your issue. But it also involves the creation of an Issue Type, which is then
+attached to an Issue Type Scheme. From there, you must create a Screen and
```
Comment (zmustapha) [inline content/articles/jira-web.md:34]: In Susan's comment, you may want to link to the Jira's documentation on Issue Type Scheme

File: content/articles/jira-web.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+attach the newly configured custom fields. Lastly, it also requires the
+configuration of an Issue Type Screen Scheme.
+
+The use of schemes and screens in Jira was something that I had to research
+quite a bit to fully understand. Fortunately there were a lot of beginner’s
+guides to Jira as well as a pretty active Jira community board for the
```
Comment (zmustapha) [inline content/articles/jira-web.md:40]: In Susan's comment, link to a beginner guide that helped you most, maybe?

File: content/articles/jira-web.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+questions that kept coming to me. One also had to be careful about which guide
+to follow, as some workflows/features are only available on Jira Cloud vs Jira
+Software.
+
+The main takeaway for me was the expanisveness of Jira and how many things
```
Comment (zmustapha) [inline content/articles/jira-web.md:45]: typo-- "expansiveness"

**Pull request merged**